app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,6 @@
+class SessionsController < ApplicationController
+
+  def new
```

Comment:
Looks like we are needing logic here still.

Reply:
since new action takes us to a form and 'current_user' is in our application_controller, we probably don't need anything in our session new action. On one line(3), we can use the syntax: " def new; end " to dry up our code.

Reply:
Yeah, I think Devin nailed it. The new action should only be routing us to a login form. The sessions create/destroy actions should contain logic for the sessions I think. Some of that is included in the development-update branch.